esp/services/ws_workunits/ws_workunitsHelpers.cpp

```diff
@@ -206,6 +206,7 @@ WsWUExceptions::WsWUExceptions(IConstWorkUnit& wu): numerr(0), numwrn(0), numinf
     }
 }
 
+#ifdef _CONTAINERIZED
```

Comment:
we should avoid not compiling container centric code, where possible.
Keeping it general and compiling in all builds, helps spot issues/has more coverage.

But also, in this case remote logs is not necessarily just a containerized feature, it could equally apply to bare-metal (if given configuration support etc.), so we should not exclude it upfront.